packages/controllers/src/controllers/ConnectionController.ts

```diff
@@ -154,7 +155,10 @@ export const ConnectionController = {
     if (!authConnector) {
       return
     }
-    await authConnector?.provider.setPreferredAccount(accountType)
+    AccountController.setPreferredAccountType(
```

Comment:
Shouldn't we call provider.setPreferredAccount to make the changes on secure site? 🤔